src/rules/selector-attribute-quotes/index.js

```diff
@@ -0,0 +1,70 @@
+import {
+  isStandardSyntaxRule,
+  isStandardSyntaxSelector,
+  parseSelector,
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+
+export const ruleName = "selector-attribute-quotes"
+
+export const messages = ruleMessages(ruleName, {
+  expectedQuotes: "Expected quotes",
```

Comment:
Can we expand these messages to include the attribute value? e.g

```js
export const messages = ruleMessages(ruleName, {
  expected: (value) => `Expected quotes around "${value}"`,
  rejected: (value) => `Unexpected quotes around "${value}"`,
})
```

[example here](https://github.com/stylelint/stylelint/blob/v7/src/rules/font-family-name-quotes/index.js#L13-L16)

I don't know if `postcss-selector-parser` has that kind of fidelity, or we might have to parse out the value ourselves.